MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -1310,12 +1310,13 @@ public void DrawPrimitives(PrimitiveType primitiveType, int vertexStart, int pri
         /// <param name="startIndex">The index within the index buffer to start drawing from.</param>
         /// <param name="primitiveCount">The number of primitives in a single instance.</param>
         /// <param name="instanceCount">The number of instances to render.</param>
+        /// <param name="baseInstance">Used to offset the instance range indexed from the instance buffer.</param>
         /// <remarks>Note that minVertexIndex and numVertices are unused in MonoGame and will be ignored.</remarks>
         [Obsolete("Use DrawInstancedPrimitives(PrimitiveType primitiveType, int baseVertex, int startIndex, int primitiveCount, int instanceCount) instead. In future versions this method can be removed.")]
         public void DrawInstancedPrimitives(PrimitiveType primitiveType, int baseVertex, int minVertexIndex,
-                                            int numVertices, int startIndex, int primitiveCount, int instanceCount)
+                                            int numVertices, int startIndex, int primitiveCount, int instanceCount, int baseInstance = 0)

```

Comment:
This one change the signature of DrawInstancedPrimitives(). It's an API change and will break existing code.
 The new functionality should be implemented as a new method.

Reply:
@nkast Resolved by maintaining original API and adding overloads.